manim/mobject/value_tracker.py

```diff
@@ -69,28 +74,58 @@ def __init__(self, value=0, **kwargs):
         self.set_value(value)
 
     def get_value(self) -> float:
-        """Get the current value of the ValueTracker. This value changes continuously when :attr:`animate` for the ValueTracker is called."""
+        """Get the current value of this ValueTracker."""
         return self.points[0, 0]
 
     def set_value(self, value: float):
         """Sets a new scalar value to the ValueTracker"""
         self.points[0, 0] = value
         return self
 
-    def increment_value(self, d_value: float):
+    def increment_value(self, d_value: Union[float, int]):
```

Comment:
```suggestion
    def increment_value(self, d_value: float]):
```
and also for the ones beneath